llvm/test/DebugInfo/X86/convert-debugloc.ll

```diff
@@ -27,7 +27,7 @@
 ; RUN:   | FileCheck %s --check-prefix=VERBOSE --check-prefix=CONV "--implicit-check-not={{DW_TAG|NULL}}"
 
 
-; SPLITCONV: Compile Unit:{{.*}} DWO_id = 0x24191746f389535f
+; SPLITCONV: Compile Unit:{{.*}} DWO_id = 0x06580df9fdd5b54b
```

Comment:
Why it changed? It doesn't test for fast-isel.

Reply:
It tests -O0 which uses fast isel

Reply:
Oh, you are correct!